test/structure-data-testing-tool.js

```diff
@@ -0,0 +1,87 @@
+const assert = require("assert");
+const { testSchema } = require("./utils");
+
+const showResult = (result) => {
+  if (result) {
+    console.log(
+      `Passed: ${result.passed.length},`,
+      `Failed: ${result.failed.length},`,
+      `Warnings: ${result.warnings.length}`
+    );
+    console.log(`Schemas found: ${result.schemas.join(",")}`);
+    // Loop over validation errors
+    if (result.failed.length > 0) {
+      console.log(
+        "⚠️  Errors:\n",
+        result.failed.map((test) => test)
+      );
+    }
+  }
+};
+
+describe("Structure data tool testing", () => {
+  const homePageSchema = ["Organization", "Website", "BreadcrumbList"];
+  it("Home page", async () => {
+    const url = "https://malibu-advanced-web.quintype.io";
+    const result = await testSchema(url, homePageSchema);
+    showResult(result);
+    assert.equal(0, result.failed.length);
```

Comment:
expecting 0 length as failed, if the test will be failed, then result.failed has values.